compose.yaml

```
@@ -21,9 +21,46 @@ services:
       - 8080:8080
       - 8443:8443
     volumes:
-      - ./dev-realm-spsh.json:/opt/keycloak/data/import/realm.json
+      - ./config/dev-realm-spsh.json:/opt/keycloak/data/import/realm.json
   redis:
     image: redis:7.2.2
     ports:
       - 6379:6379
     command: redis-server --requirepass password
+  backend:
+    profiles:
+      - full-backend
+    image: dbiam/server
+    build:
+      context: .
+    depends_on:
+      - keycloak
+      - db
+    environment:
+      NODE_ENV: dev
+      DEPLOY_STAGE: dev
+    volumes:
+      - source: 'config'
+        target: '/app/config'
+        type: bind
+  bff:
+    profiles:
+      - full-backend
+    image: dbiam/server
+    build:
+      context: .
+    depends_on:
+      - backend
+      - redis
+      - keycloak
+    restart: on-failure
+    environment:
+      NODE_ENV: dev
+      DEPLOY_STAGE: dev
+    command: ["node", "dist/src/backend-for-frontend/main.js"]
+    ports:
+      - 9091:9091
+    volumes:
+      - source: 'docker-config'
+        target: '/app/config'
+        type: bind
```

Consistent file ending
```suggestion
        type: bind
        
```